core/src/trezor/lvglui/scrs/address.py

```
@@ -499,7 +545,72 @@
+        if self.current_chain_info is None:
+            raise wire.DataError("Chain info is None")
+
+        path = self.current_chain_info["legacy_path"][:]
+        pos = self.current_chain_info["index_pos"]
+        path[pos] += self.current_index
+        self.addr_type = "Legacy"
+
+        msg_class = getattr(messages, self.current_chain_info["msg_class"])
+        return msg_class(address_n=path, show_display=False, coin_name="Bcash")
+    
+    def _prepare_xna_message(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_prepare_xna_message`

(ANN202)